vector-im/element-android
# [Devices Management] Refactor some code to improve testability (PSG-701)

## Type of change


## Content


Creating a new clean and tested `DevicesViewModel` inside the V2 package. Domain layer code has been encapsulated inside UseCases with unit tests.
All the code related to the new device management version has been move inside the V2 package.

## Motivation and context


Closes #7043 

## Screenshots / GIFs





## Tests



- Enable the feature flag for new device management
- Go to Settings -> Security&Privacy -> Show all Sessions (V2, WIP)
- Navigate into the different related screens
- Check there is no issue about the displayed info

## Tested devices

- OS version(s): Android 11

## Checklist
